lang/main.json

```diff
@@ -1,21 +1,36 @@
 {
     "addPeople": {
         "add": "Invite",
+        "addContacts": "Invite your contacts",
```

Comment:
General comment about this file. We need to get rid of the info section and put things in the right place. Let's do that in a follow-up PR.